velox/functions/prestosql/window/LeadLag.cpp

```diff
@@ -127,6 +127,9 @@ class LeadLagFunction : public exec::WindowFunction {
     }
   }
 
+  /// Handle the NULL && out range of range constant offset.
+  /// Returns true if it is NULL or out of range offset, false otherwise.
+  bool setRowNumbersForConstantNullOrOutOfRangeOffset();
```

Comment:
This method name is way too long. Please, try my suggestion instead.

Reply:
done